usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1693,35 +1777,119 @@
+                ("federal_account_name", "treasury_account__federal_account__account_title"),
                 ("program_activity_code", "program_activity__program_activity_code"),
                 ("program_activity_name", "program_activity__program_activity_name"),
                 ("object_class_code", "object_class__object_class"),
                 ("object_class_name", "object_class__object_class_name"),
                 ("direct_or_reimbursable_funding_source", "object_class__direct_reimbursable"),
                 ("disaster_emergency_fund_code", "disaster_emergency_fund__code"),
                 ("disaster_emergency_fund_name", "disaster_emergency_fund__title"),
-                ("obligations_incurred", "obligations_incurred"),
+                ("obligations_incurred", "obligations_incurred_by_program_object_class_cpe"),
+                ("obligations_undelivered_orders_unpaid_total", "obligations_undelivered_orders_unpaid_total_cpe"),
+                ("obligations_undelivered_orders_unpaid_total_FYB", "obligations_undelivered_orders_unpaid_total_fyb"),
                 (
-                    "deobligations_or_recoveries_or_refunds_from_prior_year",
-                    "deobligations_or_recoveries_or_refunds_from_prior_year",
+                    "USSGL480100_undelivered_orders_obligations_unpaid",
+                    "ussgl480100_undelivered_orders_obligations_unpaid_cpe",
+                ),
+                (
+                    "USSGL480100_undelivered_orders_obligations_unpaid_FYB",
+                    "ussgl480100_undelivered_orders_obligations_unpaid_fyb",
+                ),
+                (
+                    "USSGL488100_upward_adj_prior_year_undeliv_orders_oblig_unpaid",
+                    "ussgl488100_upward_adjust_pri_undeliv_order_oblig_unpaid_cpe",
+                ),
+                ("obligations_delivered_orders_unpaid_total", "obligations_delivered_orders_unpaid_total_cpe"),
+                ("obligations_delivered_orders_unpaid_total_FYB", "obligations_delivered_orders_unpaid_total_cpe"),
+                (
+                    "USSGL490100_delivered_orders_obligations_unpaid",
+                    "ussgl490100_delivered_orders_obligations_unpaid_cpe",
+                ),
+                (
+                    "USSGL490100_delivered_orders_obligations_unpaid_FYB",
+                    "ussgl490100_delivered_orders_obligations_unpaid_fyb",
+                ),
+                (
+                    "USSGL498100_upward_adj_of_prior_year_deliv_orders_oblig_unpaid",
+                    "ussgl498100_upward_adjust_pri_deliv_orders_oblig_unpaid_cpe",
                 ),
                 (
                     "gross_outlay_amount_fyb_to_period_end",
                     "gross_outlay_amount_fyb_to_period_end",
                 ),  # Column is annotated in account_download.py
+                ("gross_outlay_amount_fyb", "gross_outlay_amount_by_program_object_class_fyb"),
```

Comment:
Very minor but `_fyb` is not capitalized for this header name, but it is capitalized for other values.